public/locales/cn/auth.json

```diff
@@ -21,5 +21,41 @@
       "description": "没注册吗？",
       "action": "创建一个吧!"
     }
+  },
+  "signup": {
+    "title": "注册",
+    "username": {
+      "placeholder": "用户名",
+      "error": {
+        "required": "用户名是空哒!",
+        "min-length": "太短了! 至少要6位!!"
+      }
+    },
+    "email": {
+      "placeholder": "邮箱",
+      "error": {
+        "required": "请输入邮箱信息"
+      }
+    },
+    "password": {
+      "placeholder": "密码",
+      "error": {
+        "required": "你不要密码吗？"
+      }
+    },
+    "error": "要嘛不是用户名不对，就是密码不对。",
+    "user-exist": "用户已存在，请换用户名。",
```

Comment:
Imagine our backend will return different errors. We can group this type of message under `error` object.

```
{
  error: {
    existing-user.
    ...other rule,
  }
}